AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -84,9 +84,10 @@ summary_level <- opt$summary_level_vcf
 output_name   <- opt$output
 
 ## filters for gnomAD
-filter_gnomad_var    <- opt$gnomad_variable
+gnomad_variable    <- opt$gnomad_variable
 filter_variant_depth <- opt$variant_depth
 filter_variant_af    <- opt$variant_af
+filter_gnomad_af    <- opt$gnomad_af
```

Comment:
I don't think you need these chunks actually - you can use `opt$XXX` within the code below

Reply:
I have added this for readability purposes and to make debugging easier. At times to debug, we need to manually change and run the code in the script as opposed to on the command line. I prefer to have this for now especially as we finalize/modify the code.